Dockerfile

```diff
@@ -49,6 +49,8 @@ EXPOSE 5432
 
 ADD root /
 
+RUN mv /usr/share/container-scripts/postgresql/help.1 /help.1
+
```

Comment:
Below, you create the file `root/usr/share/container-scripts/postgresql/help.1`, why not create `root/help.1` directly rather than creating it elsewhere and then moving it?